### ISSUE TITLE
Question: How do I generate a validation set .bin file with my own data?

### ISSUE THREAD

**Issue by crownleo** [luckycallor/InsightFace-tensorflow#20]
Looking forward to your reply

**Comment by billamiable** [comment]
i think i might found an answer here https://github.com/deepinsight/insightface
